cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/PreferencesHttpHandlerTest.java

```diff
@@ -33,6 +33,7 @@
  * Tests for {@link PreferencesHttpHandler}
  */
 public class PreferencesHttpHandlerTest extends AppFabricTestBase {
+
```

Comment:
There are tests that test those endpoints. setProperty, getProperty, deleteProperty are helper methods that help construct the right URI.